### DISCUSSION TITLE
ddos

### DISCUSSION THREAD

**Discussion by zszs2023** [wazuh/wazuh#16891] [Q&A]
hello 
please, how can i configure wazuh to detect ddos attacks

**Comment by mdiego92** [comment]
Hi @zszs2023,

To detect ddos attacks I would recommend integrating [Suricata](https://suricata.io) with Wazuh. Please find the required implementation steps in the following documentation:

[Wazuh Suricata Documentation
](https://documentation.wazuh.com/current/learning-wazuh/suricata.html)[Network IDS integration - Proof of Concept guide · Wazuh documentation](https://documentation.wazuh.com/current/proof-of-concept-guide/integrate-network-ids-suricata.html) 
[Detection Attack using Suricata](https://scribe.nixnet.services/@mshulkhan/detection-attack-using-suricata-1-5ea7b2f62551)

Please let me know if you have any remaining questions!

**Reply by zszs2023** [reply]
thanks a lot for your helping
please, how can i use wazuh as central tools to monitor social media 
thanks a lot
